feature-utils/poly-look/test/react-components/filterChips.test.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { render, fireEvent } from "@testing-library/react";
+import "@testing-library/jest-dom";
+import FilterChips from "../../src/react-components/filterChips/filterChips.jsx";
+
+const chipsContent = [...Array(3).keys()].map((i) => `chip${i}`);
+const mockedHandleClick = jest.fn();
+const allChipContent = "All";
+const firstChipContent = chipsContent[0];
+
+const component = (
```

Comment:
`chipsContent` can be either a string array or an object array. I think you're testing only the first option, not the second. The tests should check also if defaults in that case work correctly